cw_core/lib/receive_page_options.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:bitcoin_base/bitcoin_base.dart';
+
+class CoreReceivePageOption {
+  static const mainnet = CoreReceivePageOption._('mainnet');
+  static const anonPayInvoice = CoreReceivePageOption._('anonPayInvoice');
+  static const anonPayDonationLink = CoreReceivePageOption._('anonPayDonationLink');
+
+  const CoreReceivePageOption._(this._value);
+
+  final String _value;
+}
+
+const ReceivePageOptions = [
+  CoreReceivePageOption.mainnet,
+  CoreReceivePageOption.anonPayInvoice,
+  CoreReceivePageOption.anonPayDonationLink
+];
+
+class BitcoinReceivePageOption extends CoreReceivePageOption {
```

Comment:
**Enhancement:**
better to be moved to `cw_bitcoin` and thus no need to add `bitcoin_base` dependency to `cw_core`